app/main/auth.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+from flask import abort
+from flask_login import current_user
+
+
+def role_required(*roles):
+    """Ensure that logged in user has one of the required roles.
+
+    Return 403 if the user doesn't have a required role.
+
+    Should be applied before the `@login_required` decorator:
+
+        @login_required
+        @role_required('admin', 'admin-ccs')
```

Comment:
I'm not sure about the name either, but couldn't come up with anything better in the API PR. It's readonly right now, but that's more likely to change than the "CSS" part, which would make `admin-readonly` much more misleading.